lib/src/health.dart

```diff
@@ -57,6 +57,8 @@ Health calcHealth({
     }
   }
 
+  final analyzerSuggestions = <_Suggestion>[];
```

Comment:
Instead of a one-off special purpose class, you could have a map from the Suggestion to the percentage?

Reply:
We'll still need to sort and process them in order, I think a Map would make that slightly more complex.